index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <meta charset="UTF-8" />
+  <meta name="author" content="Uchenna and Iris" />
+  <meta name="viewport" content="width=device-width, initial-size=1.0" />
+  <meta name="keyword" content="SignUp clone" />
+  <link rel="stylesheet" href="assets/css/style.css" />
+  <script src="https://use.fontawesome.com/018e0d5e7a.js"></script>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Comic+Neue:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+    rel="stylesheet"
+  />
+  <title>Mint Sing-up Page</title>
+
+  <body>
+    <header>
+      <nav>
+        <div>
+          <p>Already have an Intuit account?</p>
+          <a
+            href="https://accounts.intuit.com/account-recovery.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D05e5f79c-e029-4997-a5a2-0e39e8c3a1b1%26adobe_mc%3DMCMID%253D01118330359371472344012076105835982965%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1600362777%26ivid%3D05e5f79c-e029-4997-a5a2-0e39e8c3a1b1"
+            target="blank"
+            >I forgot my User or Password</a
+          >
+        </div>
+      </nav>
+      <button type="submit" class="_button">
+        <i class="fa fa-lock"></i> &nbsp; Sign In
+      </button>
+    </header>
+    <main>
+      <div>
+        <div class="logo">
+          <div class="intuit">
+            <img src="assets/images/intuit-logo.PNG" alt="intut logo" />
+          </div>
+          <picture>
+            <img src="assets/images/mint-logo.PNG" alt="logo" class="mint" />
+          </picture>
+          <picture>
+            <img src="assets/images/quickbooks-logo.PNG" alt="logo" />
+          </picture>
+          <picture>
+            <img
+              src="assets/images/logo-turbotax.PNG"
+              alt="logo"
+              class="turbo"
+            />
+          </picture>
+        </div>
```

Comment:
It shows the Mint logo and text (logo on the left, text on the right) 

 It shows the Quickbooks logo and text (logo on the left, text on the right

 It shows the Turbotax logo and text (logo on the left, text on the right)